recipes/stripe.rb

```diff
@@ -117,6 +117,14 @@
   action :create
 end
 
+# Make sure that the permissions for the 'mysql' directory are set correctly.
+# When recovering from a backup uids could have changed.
+execute "change permissions #{new_mysql_dir} owner" do
+  command "chown --recursive --silent mysql:mysql #{new_mysql_dir}"
+  not_if "stat -c %U #{new_mysql_dir}/mysql |grep mysql"
```

Comment:
Instead of going out and running shell commands let's use a native ruby command like:
`only_if { Etc.getpwuid(File.stat('/opt/foo').uid).name != 'mysql' }`